test/acceptance/features/unbindAppToService.feature

```diff
@@ -106,3 +106,50 @@ Feature: Unbind an application from a service
 
         Then The env var "BACKEND_HOST" is not available to the application
         And The env var "BACKEND_USERNAME" is not available to the application
+    
+    Scenario: Unbind a generic test application from the backing service when the test application has been deleted
```

Comment:
Does that mean that service binding gets removed by deleting the application from the cluster? What happens when we first create service binding and then add application to the cluster?

Reply:
no,  as described in the PR description,  in normal case, the application deleting won't introduce any error of unbind.   Also, if sbr created before the application,  the sbr won't have issue either. It will just reconcile until the application is ready. 

But problem that addressed in this PR handles a race condition, in which the sbr unbind reconcile and application deletion happens at nearly the same time (i.e namespace is deleted),  then unbind flow is blocked with `application not found` error in previous implementation.

Reply:
@cdlliuy thanks for the clarification. How about that we rephase Scenario to match better to the above explanation, something like:

```
Remove binding when application does not exist anymore
@pedjak 
```

Reply:
@pedjak , thanks , updated the description